free_fleet_adapter/free_fleet_adapter/nav1_robot_adapter.py

```diff
@@ -49,11 +49,14 @@
 
 class Nav1TfHandler:
 
-    def __init__(self, robot_name, zenoh_session, tf_buffer, node):
+    def __init__(self, robot_name, zenoh_session, tf_buffer, node, \
```

Comment:
back slashes are not required within brackets, this applies to the other changes too